components/automate-chef-io/content/docs/users.md

```diff
@@ -105,8 +105,8 @@ curl -X DELETE -H "api-token: $TOK" -H "Content-Type: application/json" https://
 ### User Self-Maintenance
 
 Local Automate users can manage their own name and password through the Chef Automate user interface.
-Select your full name in the top navigation bar,
-then select **Your Profile** from the drop-down.
+Click the user icon in the top navigation bar,
```

Comment:
We've removed the word "Click" from the docs as it is centered on people who use pointing devices. Could we keep this as _Select_ here?

Reply:
Seconded.